aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -152,6 +152,17 @@ module std::features {
         is_enabled(BLS12381_BASIC_OPERATIONS)
     }
 
+    /// Whether native_public_key_validate aborts when a public key of the wrong length is given
+    /// Lifetime: ephemeral
+    const PUBKEY_VALIDATE_RETURN_FALSE_WRONG_LENGTH: u64 = 13;
+
+    public fun get_pubkey_validate_returns_false_wrong_length(): u64 { PUBKEY_VALIDATE_RETURN_FALSE_WRONG_LENGTH }
```

Comment:
Do we need explicit functions here? Is this feature flag going to be used in Move?

Reply:
We do not. This is used only in the native.

But I assume we should leave the constant here reserved.

Reply:
We can leave the constant, but let's remove the functions as they're not used